lib/editor/components/pattern/EditSettings.js

```diff
@@ -75,6 +79,25 @@ export default class EditSettings extends Component<Props, State> {
     const noSegmentIsActive = !patternSegment && patternSegment !== 0
     return (
       <div>
+        <ControlLabel><small>Snap to options</small></ControlLabel>
```

Comment:
Nitpick: should we shorten this string?
```suggestion
        <ControlLabel><small>Snap to</small></ControlLabel>
```